data/themes.yml

```diff
@@ -1,3 +1,4 @@
+- { colors: '#F1F3F5,#DAD8DA,#D3DFE3,#303030,#C2E2ED,#303030,#1682FB,#34C749', name: 'A Similar to macOS' }
```

Comment:
Can you please rename this to `macOS` instead and add it to the end of the file? Thanks.